test/xspec.bats

```diff
@@ -190,3 +190,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "${lines[3]}" =~ "Testing with" ]]
 }
+
+@test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file:///home/travis/build/xspec/xspec/ -oresult=/home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc && grep 'charset=UTF-8' /home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html"
```

Comment:
Since the report is XHTML and Saxon is readily available, we might want to have Saxon do `grep`'s filtering job and inspect its output in Bats. For example,

```winbatch
P:\xspec>bin\xspec.bat tutorial\escape-for-regex.xspec
...
passed: 5 / pending: 0 / failed: 1 / total: 6
Report available at P:\xspec\tutorial\xspec\escape-for-regex-result.html
Done.

P:\xspec>java -cp "%SAXON_CP%" net.sf.saxon.Query -s:"tutorial\xspec\escape-for-regex-result.html" -qs:"declare default element namespace 'http://www.w3.org/1999/xhtml'; string(html/head/meta[@http-equiv eq 'Content-Type']/@content)" !method=text
text/html; charset=UTF-8
```